app/misc/git.ts

```diff
@@ -1,8 +1,10 @@
 import * as nodegit from "git";
 import NodeGit, { Status } from "nodegit";
+import * as simplegit from 'simple-git/promise';
```

Comment:
I don't know how to feel about adding `simple-git` to the project.  I don't like the idea of mixing different libraries together to do the same thing, in this case git functionality.  I understand the necessity for it, as `node-git` did not have what you needed.  I still wonder whether it is wise to introduce a second package.

For the purposes of this project it is probably fine, and it works well so I can't complain!

Reply:
> I don't know how to feel about adding `simple-git` to the project. I don't like the idea of mixing different libraries together to do the same thing, in this case git functionality. I understand the necessity for it, as `node-git` did not have what you needed. I still wonder whether it is wise to introduce a second package.
> 
> For the purposes of this project it is probably fine, and it works well so I can't complain!

I agree with @nikola-rados; however, simple-git has MORE functionality. So going forward we should STOP using nodegit and use simple-git. In fact, we should probably create tickets to start replacing nodegit with simplegit...